phpBB/adm/style/acp_main.html

```diff
@@ -95,112 +95,116 @@ <h3>{L_ERROR_MBSTRING_HTTP_OUTPUT}</h3>
 	<!-- EVENT acp_main_notice_after -->
 
 		<div class="lside">
-			<table class="table1 zebra-table no-header" data-no-responsive-header="true">
+			<table class="table2 zebra-table no-header" data-no-responsive-header="true">
 				<thead>
 					<tr>
-						<th>{L_STATISTIC}</th>
-						<th>{L_VALUE}</th>
+						<th>{{ lang('STATISTIC') }}</th>
+						<th>{{ lang('VALUE') }}</th>
 					</tr>
 				</thead>
+
 				<tbody>
 					<tr>
-						<td class="tabled">{L_NUMBER_POSTS}{L_COLON}</td>
-						<td class="tabled"><strong>{TOTAL_POSTS}</strong></td>
+						<td class="tabled">{{ lang('BOARD_STARTED') ~ lang('COLON') }}</td>
```

Comment:
why you are switching the order of two tables around? This makes for a lot of unnecessary diffs

Reply:
I brought back the original arrangement of the elements, as in fact they are currently in 3.1.x / 3.2.x. The arrangement changed in https://github.com/phpbb/phpbb/pull/5187 and nobody noticed it.

---
3.1.x

![immagine](https://user-images.githubusercontent.com/480857/71449395-ecf91a00-274b-11ea-8797-5e3456b99ce6.png)
---

3.2.x

![immagine](https://user-images.githubusercontent.com/480857/71449399-fedabd00-274b-11ea-8d69-031aa4d59cb4.png)